org2elcomment.el

```diff
@@ -3,7 +3,6 @@
 ;; Copyright (C) 2016  Junpeng Qiu
 
 ;; Author: Junpeng Qiu <qjpchmail@gmail.com>
-;; Package-Requires: ((org "8.3.4"))
```

Comment:
Why? org is on GNU ELPA and I think the export mechanism depends on an `org` version at least `8.x.x`. This line should be fine here